packages/examples/phases/src/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React, {FC, Suspense, useEffect} from 'react';
+import ReactDOM from 'react-dom';
+
+import ModulePhase1 from './phase1';
+import {deferredLoadComponent} from './utils';
+const Phase2 = deferredLoadComponent(
+  importDeferredForDisplay<typeof import('./phase2')>('./phase2'),
```

Comment:
The type here will need to be enforced with linting. If we add an autofixer, it should be pretty easy to use for devs.